test/unit/extension/lib/bigcommerce/CartRepositoryTest.js

```diff
@@ -106,4 +106,28 @@ describe('BigCommerceCartRepository - unit', function () {
 
     return subjectUnderTest.getCheckoutUrl().should.eventually.be.rejectedWith(Error)
   })
+
+  it('should properly call bigCommerce client delete method when doing a deleteProductsFromCart', function () {
```

Comment:
Maybe we could make a test that is able to show that every item id passed inside of array will be deleted, not just one.

Reply:
I modified test that tests successful delete from cart so it deletes two products. Utilized onCall() to check that the bigCommerce client delete method is called twice with the correct parameters based on the item being deleted.